.eslintrc.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     settings: {
         react: {
             version: (() => {
-                const mallardPackage = require('./projects/Mallard/package.json')
+                const mallardPackage = require('./projects/Apps/editionsSrc/package.json')
```

Comment:
camel case directory names feels a bit strange? I have no strong opinions though!

Reply:
Happy to change it?